include/ignition/gui/qml/GzPose.qml

```diff
@@ -124,6 +142,16 @@ Item {
     }
   }
 
+  /**
+   * Used to create a plotting icon
+   */
+  Component {
+    id: gzPlotPose
+    GzPlot {
```

Comment:
Can we make the plot icon optional? This way the widget can be used in plugins that don't have plottable data, like `GridConfig`, `ViewAngle`, etc.